### ISSUE TITLE
Fine tuning on sintel. 

### ISSUE THREAD

**Issue by TusharNimbhorkar** [lmb-freiburg/flownet2#141]
Hi,

I am trying to fine tune Flownet2-S on sintel. But In https://github.com/lmb-freiburg/flownet2/issues/127#issuecomment-381321198(url). You say the model is fine-tuned on the mix of Final and clean. Do you mean like fine tuning on clean+final(which will be around 2k image pairs)? Or some sequences from clean and some sequences from final(around 1k image pairs. By keeping the sequences unique). 

Also, I am wondering the weights given in Flownet2-S. Are they already finetuned on sintel or only trained on Flying chairs? 

Thanks.

**Comment by nikolausmayer** [comment]
Hi,

the following scenes are reserved for a validation set:
`ambush_6`, `bamboo_2`, parts of `cave_4`, `market_6`, parts of `temple_2`.

The rest is used as finetuning training data. AFAIK we treat clean and final the same.

Only `*-sintel` or `*-ft` networks are trained on anything but FlyingThings/Chairs.

**Comment by TusharNimbhorkar** [comment]
Thanks. Closing the issue
